easybuild/easyconfigs/r/RepeatMasker/RepeatMasker-4.0.6-foss-2016a-Perl-5.22.1.eb

```diff
@@ -0,0 +1,42 @@
+easyblock = 'PackedBinary'
+
+name = 'RepeatMasker'
+version = '4.0.6'
+versionsuffix = '-Perl-%(perlver)s'
+
+homepage = 'https://bitbucket.org/fenics-project/fiat'
+description = """RepeatMasker is a program that screens DNA sequences for interspersed repeats and low complexity DNA sequences.
+ The output of the program is a detailed annotation of the repeats that are present in the query sequence as well as a modified 
+ version of the query sequence in which all the annotated repeats have been masked (default: replaced by Ns). Currently over 56%
+ of human genomic sequence is identified and masked by the program. Sequence comparisons in RepeatMasker are performed by one of
+ several popular search engines including nhmmer, cross_match, ABBlast/WUBlast, RMBlast and Decypher. RepeatMasker makes use of
+ curated libraries of repeats and currently supports Dfam ( profile HMM library derived from Repbase sequences ) and Repbase, a
+ service of the Genetic Information Research Institute."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['http://www.repeatmasker.org/']
+sources = ['%%(name)s-open-%s.tar.gz' % '-'.join(version.split('.'))]
+
+dependencies = [
+    ('Perl', '5.22.1'),
+    ('TRF', '4.09', '-linux64',('dummy', 'dummy')),
+    ('RMBlast', '2.2.28', '-Python-2.7.11'),
+]
+
+skipsteps = ['configure', 'test']
+
+# RepeatMasker has a configure that needs some input, if HMMER is preferred as search engine include HMMER as dependency and
+# replace '2' with '4': 
+postinstallcmds = [
+  "cd %(installdir)s && \
+   printf '\n\n\n\n%s\n%s\n\n%s\n' '2' $EBROOTRMBLAST '5' > conf.in && \
+   ./configure < conf.in && rm conf.in",
```

Comment:
@RvDijk there's no cleaner way to handle this? Does the `configure` accept arguments directly, or does it maybe take a file with the setting in it?

For interactive installers, we usually resort to an easyblock rather than 'trickery' like this in easyconfig files...

There, we could auto-detect whether `HHMER` is used as a dependency, and act accordingly.